src/components/organisms/PaymentModal/PaymentModal.tsx

```diff
@@ -1,93 +0,0 @@
-import React, { useState } from "react";
-import { Modal } from "react-bootstrap";
-
-import { AnyVenue, VenueEvent } from "types/venues";
-
-import useConnectUserPurchaseHistory from "hooks/useConnectUserPurchaseHistory";
-import { hasUserBoughtTicketForEvent } from "utils/hasUserBoughtTicket";
-import { isUserAMember } from "utils/isUserAMember";
-import { Purchase } from "types/Purchase";
-import "./PaymentModal.scss";
-import PaymentForm from "./PaymentForm";
-import PaymentConfirmation from "./PaymentConfirmation";
-import { useUser } from "hooks/useUser";
-import { useSelector } from "hooks/useSelector";
-import { WithId } from "utils/id";
-import {
-  currentVenueSelectorData,
-  userPurchaseHistorySelector,
```

Comment:
Does `userPurchaseHistorySelector` still exist in `utils/selectors`? If so, it should be removed, along with the types in `src/types/Firestore.ts` + clean up any related code that still references them in any way.

Reply:
fixed in ebe967cd50437f138f6685ec3ce64daef7abeb40